extension/chrome/elements/compose.ts

```diff
@@ -199,6 +199,11 @@ export class ComposeView extends View {
     this.S.cached('body').on('focusin', setActiveWindow);
     this.S.cached('body').on('click', setActiveWindow);
     this.S.cached('icon_help').click(this.setHandler(async () => await this.renderModule.openSettingsWithDialog('help'), this.errModule.handle(`help dialog`)));
+    this.S.cached('input_text').on('keypress paste',this.setHandler((el, ev) => {
+      if (this.S.cached('input_text').children('div').text().length >= 10) { // setting 10 as a limit for ease of testing; 50kB is tested to work with low spec pc;
+        ev.preventDefault();
+      }
+    }));
```

Comment:
Good afternoon, sir, @rrrooommmaaa. I'm trying to add an input limit on the secure compose message box and it doesn't work as expected; 

What happens is that onpaste seems not working and doesn't prevent the user from pasting text/contents even though the keypress works just fine.

I tried to try the whole concept at https://jsfiddle.net/ftye9xrn/ starting on the minimal code. The output is much more acceptable there. Do you have any suggestions here?

Furthermore, the goal of this PR is to prevent user's from populating input fields with large texts that cause an unresponsive browser tab (not the entire browser)  - related PR https://github.com/FlowCrypt/flowcrypt-security/issues/206. If by any chance you have any feedback or suggestions, please let me know and I'll be happy to share them with Tom.

Thank you!

Reply:
> I tried to try the whole concept at https://jsfiddle.net/ftye9xrn/ starting on the minimal code. The output is much more acceptable there. Do you have any suggestions here?

Hi @martgil. Sorry for the late answer, I somehow missed the notification.
The `paste` event handler obviously takes the value of the input _before_ the modification, this is why there is no reaction.
The handler for paste event should look somthing like this:
```
$("#test").on("paste", function (e) {
  const selectedLength = window.getSelection().toString().length;
  const curLength = $("#test").text().length;
  const clipboardLength = e.originalEvent.clipboardData.getData('text').length;
  const resultingLength = curLength - selectedLength + clipboardLength;
  if (resultingLength > 10) {
  	e.preventDefault();
  }
})
```
However the [Selection](https://developer.mozilla.org/en-US/docs/Web/API/Selection) is a complex object, consisting of possibly more than one range, maybe we need to do something more sophisticated than `toString().length` on it, I'm encouraging you to further investigate this, @martgil

Reply:
Probably it's better to use these properties:
- `input.selectionStart` – position of selection start (writeable),
- `input.selectionEnd` – position of selection end (writeable),
as described [here](https://javascript.info/selection-range#selection-in-form-controls)

Reply:
thank you for providing such detailed feedback, sir. I will get back to this after i accomplished some tasks with higher priority. thanks again.